status/chat.nim

```diff
@@ -335,6 +335,12 @@ proc processMessageUpdateAfterSend(self: ChatModel, response: string): (seq[Chat
   if chats.len == 0 and messages.len == 0:
     self.events.emit("messageSendingFailed", Args())
     return
+
+  # This fixes issue#3490
```

Comment:
nit: rather than a comment pointing to another repository issue (hard to find)
What about a private proc which explain why we are doing that:
```
proc findMessageAfterReplySend(messages):
  for m in messages:
    if(m.responseTo.len > 0):
      return messages
```

Even if not adding the proc at least a `break` would be good

Reply:
No problem a proc or a comment are same thing. But in that case you need to fix the link to the issue as we are in status-lib project and you are referring another project issue

Reply:
I am not aware that we have separate space for `status-lib` part?

Reply:
We don't, you know it, I know it but this is a public repository, anyone can read the code, the least we can do is make it correct IMO